src/plugins/analytics.js

```diff
@@ -0,0 +1,54 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment'
+
+export default (function () {
+  if (!ExecutionEnvironment.canUseDOM) {
+    return null
+  }
+
+  // Emit events for all github link clicks
+  function anchorListener(event) {
+    let element = event.target
+    if (
+      element.tagName == 'A' &&
+      element.href.startsWith('https://github.com')
+    ) {
+      analytics.track('Github Navigation', {
+        type: document.title.split(' | ')[0],
```

Comment:
Can this return empty?

Reply:
No, docusaurus always pipes the title of the doc with the title from the docusaurus config